lib/deepsort.rb

```diff
@@ -86,7 +86,7 @@ def deep_sort_by!(&block)
           value
         end]
 
-      end.sort_by(&block)])
+      end.sort_by!(&block)])
```

Comment:
isn't the in-place operator covered by the closing replace() here?

Reply:
yes, but this way we produce 1 less throwaway object